### PR TITLE
chore: Update test-kustomize-build in .prow.yaml

### PR DESCRIPTION
**Pull request by tumido**
Part of: https://github.com/operate-first/toolbox/issues/18

With https://github.com/operate-first/toolbox/pull/13 we have the kustomize-build in toolbox now

/cc @goern @harshad16

### CODE REVIEW

**Review by harshad16 — APPROVED** [review]
/lgtm

File: .prow.yaml [hunk 1]
```diff
@@ -8,9 +7,9 @@ presubmits:
     context: aicoe-ci/prow/kustomize-build
     spec:
       containers:
-        - image: quay.io/operate-first/opf-toolbox:v0.2.0
+        - image: quay.io/operate-first/opf-toolbox:v0.3.1
           command:
-            - "./test-kustomize-build"
+            - "test-kustomize-build"
```
Comment (harshad16) [inline .prow.yaml:12]: is this to run from the image? if so, isn't the file kept at /usr/local/bin/ ?

**Review by harshad16 — COMMENTED** [review]
i just noticed this https://github.com/operate-first/toolbox/blob/cf04bd90a336eceb95e96b1f3362b45fb29c4954/Dockerfile#L49

File: .prow.yaml (hunk 1 above)
Comment (tumido) [reply .prow.yaml:12]: resolved in `v0.3.2` - can be called directly

**Review by harshad16 — APPROVED** [review]
/lgtm

**Review by HumairAK — APPROVED** [review]

**Pull request merged**